tests/zfs-tests/include/libtest.shlib

```diff
@@ -134,6 +134,19 @@ function is_dilos
 	fi
 }
 
+# Determine platform name
+#
+# Return platform name
+
+function platform_name
+{
+	if is_dilos; then
+		echo "dilos"
+	else
+		echo $(uname)
```

Comment:
Technically not needed here, but it seems prudent to quote this anyway.  Out of curiosity, what does `uname` return on dilos?

`echo "$(uname)"`

Reply:
the same with illumos:
```
u1@zfs-t3:~$ uname
SunOS
```